cli/script/command-executor.ts

```diff
@@ -703,8 +703,13 @@ function printDeploymentHistory(command: cli.IDeploymentHistoryCommand, deployme
                 if (releaseSource) {
                     releaseTime += "\n" + chalk.magenta(`(${releaseSource})`).toString();
                 }
-
-                var row = [packageObject.label, releaseTime, packageObject.appVersion, packageObject.isMandatory ? "Yes" : "No"];
+                
+                var displayLabel: string | Chalk.ChalkChain = packageObject.label;
+                if (packageObject.isDisabled) {
+                    displayLabel = chalk.red("(x)") + packageObject.label;
```

Comment:
chalk.red(...).toString() would remove your need to have type `Chalk.ChalkChain` above.